src/styles/products.css

```diff
@@ -0,0 +1,35 @@
+:root {
+  --product-color: var(--token-color-hashicorp-brand);
```

Comment:
It might be helpful to start considering a [version of dev dot 'product meta' ](https://github.com/hashicorp/web-platform-packages/tree/main/packages/product-meta)for situations like this. Not totally attached to this approach but could see how something like it would be helpful.

Reply:
@zchsh @ashleemboyer i'm curious to get your thoughts on this spike.

Reply:
LGTM! 👍🏻 Would probably be useful in `IconCardLink` as well. FWIW the current product context is pretty much the dev dot version of "product meta". One thing it doesn't have is `themeClass` but that's easy to add!

Reply:
Oh cool, that seems like a good ticket for a future improvement! I'll write one up.

Reply:
https://app.asana.com/0/1202097197789424/1202428521623295